ddtrace/tracer/rules_sampler.go

```diff
@@ -271,6 +271,17 @@ func NameServiceRule(name string, service string, rate float64) SamplingRule {
 	}
 }
 
+func NameServiceResourceRule(name, service, resource string, rate float64) SamplingRule {
+	return SamplingRule{
+		Service:  globMatch(service),
+		Name:     globMatch(name),
+		Resource: globMatch(resource),
+		Rate:     rate,
+		ruleType: SamplingRuleTrace,
+		globRule: &jsonRule{Name: name, Service: service, Resource: resource},
+	}
+}
+
```

Comment:
The PR looks good to me, however i would refrain from addin this option as we are removing them in V2 anyways. No need to add to public API for the sake of the internal function, when there are other ways to achieve the same things 🙌

Reply:
what do you recommend?

Reply:
Remove this option and use the regular Sampling rule struct.

Reply:
done.